applications/dashboard/library/class.nestedcollection.php

```diff
@@ -503,20 +503,20 @@ public function removeItem($key) {
      * @param array $item The item to generate CSS class for.
      * @return string The generated CSS class.
      */
-    private function buildCssClass($prefix, $item) {
+    private function buildCssClass($prefix, array $item = []) {
```

Comment:
I have high confidence in this change not breaking anything because it is a private method and I've verified all of the calls and the inputs passed.